examples/rcbilling-demo/src/tests/main.test.ts

```diff
@@ -31,6 +31,27 @@ test.describe("Main", () => {
     );
   });
 
+  test("Displays specific offering when offering_id is provided", async ({
```

Comment:
I think this test makes sense only if we actually use the new method (through the `usePurchasesLoaderData` hook) as mentioned above, otherwise is not really testing the new functionality we added.